sound/soc/intel/common/soc-acpi-intel-rpl-match.c

```diff
@@ -350,6 +359,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_rpl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-rpl-rt711.tplg",
 	},
+	{
+		.link_mask = 0x4, /* link2 required */
+		.links = rplp_crb,
+		.drv_name = "sof_sdw",
+		.sof_tplg_filename = "sof-rpl-rt711-sdw2.tplg",
```

Comment:
Maybe "sof-rpl-rt711-l2.tplg" to align with others? And, please edit the "sof-rpl-rt711.tplg" name at line 360, too.